.drone.yml

```diff
@@ -492,6 +492,39 @@ steps:
     HOST: grafana-server
   image: cypress/included:9.5.1-node16.14.0-slim-chrome99-ff97
   name: end-to-end-tests-various-suite
+- commands:
+  - apt-get update && apt-get install git curl jq --yes
+  - cd ..
+  - git clone "https://$${GITHUB_TOKEN}@github.com/grafana/cloud-data-sources" --depth=1
+  - cd cloud-data-sources/e2e-scripts
+  - ./scripts/install-dependencies-ci.sh azure
```

Comment:
maybe we can skip the `cd`, I don't think there is a problem to clone the repo in the current working directory (and call the scripts as `./cloud-data-sources/e2e-scripts/scripts/install-dependencies-ci.sh`)

Reply:
I was doing this as `yarn` was having issues if the repo was cloned into the Grafana directory, but I can make this work 😊